images/base/ubuntu.Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN apt-get update && \
     pipx \
     python3 \
     python3-pip \
-    # software-properties-common \
+# software-properties-common \
```

Comment:
This does let us build the image for armv7 but will break the usage of `add-apt-repository` for other archs.

Reply:
Can we conditionally disable it only for armv7 arch?